README.md

```diff
@@ -1,37 +1,42 @@
-# Getting started
```

Comment:
You trashed the whole documentation about how to contribute... why ?

You could've just put it at the bottom